src/components/ebay-table/component.ts

```diff
@@ -129,6 +145,28 @@ export default class EbayTable extends Marko.Component<Input, State> {
         });
     }
 
+    setLoading() {
+        if (this.input.bodyState === "loading") {
+            if (this.tbody) {
+                requestAnimationFrame(() => {
+                    focusables(this.tbody).forEach((focusable: HTMLElement) => {
+                        if (!focusable.getAttribute("disabled")) {
```

Comment:
Focusables should not return elements that are disabled. This additional check is unnecessary IMO.  I would leave it up to you if you want to keep code as is or use makeup to fix when an interactive element not under makeup list is being used.